### DISCUSSION TITLE
Is OverlayScrollbars v2 stable?

### DISCUSSION THREAD

**Discussion by AgentSmith0** [KingSora/OverlayScrollbars#416] [Q&A] [answered]
Is OverlayScrollbars v2 stable?

**Comment by KingSora** [comment, marked as answer]
Good day @AgentSmith0 
Its no stable yet, still working on it!

**Reply by AgentSmith0** [reply]
Alright, thank you very much for this awesome library!

**Reply by AgentSmith0** [reply]
Any updates to the future to this library? I like it very much...
Would you recommend using v2 or should I wait for a stable version?

**Reply by AgentSmith0** [reply]
@KingSora Do you have an idea when v2 will be stable?

**Reply by KingSora** [reply]
I'll try to make it till mid of the year, but v2 isn't stable currently.

**Reply by AgentSmith0** [reply]
That would be awesome... Thank you so much!
